client/my-sites/customer-home/cards/tasks/renew/index.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_RENEW_EXPIRED_PLAN } from 'calypso/my-sites/customer-home/cards/constants';
+import expiredIllustration from 'calypso/assets/images/customer-home/disconnected-dark.svg';
+import expiringIllustration from 'calypso/assets/images/customer-home/disconnected.svg';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getSite } from 'calypso/state/sites/selectors';
+import { getSitePurchases } from 'calypso/state/purchases/selectors/get-site-purchases';
+import { withLocalizedMoment } from 'calypso/components/localized-moment';
+
+const Renew = ( { card, moment, purchases, site, siteSlug } ) => {
+	const translate = useTranslate();
+	const hasExpired = card === TASK_RENEW_EXPIRED_PLAN;
+
+	const planPurchase = purchases.find(
+		( purchase ) => purchase.productId === site?.plan.product_id
+	);
+
+	const planName = site?.plan.product_name_short;
+	const planSlug = planPurchase?.productSlug;
+	const expiry = moment( planPurchase?.expiryDate );
+	const expiryToday = Math.abs( moment().diff( expiry, 'hours' ) ) < 24;
+	const expiryText = expiryToday ? expiry.format( '[today]' ) : expiry.fromNow();
+	const isOwner = site?.plan.user_is_owner;
+
+	const title = hasExpired
+		? translate( 'Reactivate your %(planName)s plan', { args: { planName } } )
+		: translate( '%(planName)s plan expiring soon', { args: { planName } } );
+	let description;
+	let actionText;
+	if ( isOwner && hasExpired ) {
+		description = translate(
+			'Your %(planName)s plan expired %(timeSinceExpiry)s. Reactivate now to continue enjoying features such as increased storage space, access to expert support, and automatic removal of WordPress.com ads.',
```

Comment:
Just noting that yes, these listed features do apply to every plan (I didn't realize they also applied to our cheapest personal plans).